docs/example-scenario/ai/movie-recommendations.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Movie recommendations on Azure
+description: Use  machine learning to automate movie, product, and other recommendations using machine learning and an Azure Data Science Virtual Machine (DSVM) to train a model on Azure.
+author: njray
+ms.date: 12/7/2018
+ms.custom: product-team
+---
+
+# Movie recommendations on Azure
+
+This example scenario shows how a business can use machine learning to automate product recommendations to their customers.  An Azure Data Science Virtual Machine (DSVM) is used to train a model on Azure that recommends movies to users based on the current ratings users have given to movies.
+
+Recommendations can be useful in various industries from retail to news to media. Potential applications include providing  product recommendations in a virtual store, providing news or post recommendations, or providing music recommendations.  Traditionally, businesses had to hire and train assistants to make personalized recommendations to customers. Today, we can  provide customized recommendations at scale by utilizing Azure to train models to understand customer preferences.
+
+## Relevant use cases
+
+Consider this scenario for the following use cases:
+
+* Movie recommendations on a website.
+* Consumer product recommendations in a mobile app.
+* News recommendations on streaming media.
+
+## Architecture
+
+![Architecture of a machine learning model for training movie recommendations][architecture]
+
+This scenario covers the training and evaluating of the machine learning model by using the Spark [alternating least squares][als] (ALS) algorithm on a dataset with ratings of movies. The steps for this scenario are as following:
+
+1. The front-end website or app service collects historical user-movie interaction data, which are usually represented in a table of user, item, and numerical rating tuples.
+2. The collected historical data is stored in a blob storage.
+3. A DSVM is often used to experiment with or productize a Spark ALS recommender model. The ALS model is trained using a training dataset, which is produced from the overall dataset by applying the appropriate data splitting strategy. For example, the dataset can be split into sets randomly, chronologically, or stratified, depending on the business requirement. Similar to other machine learning tasks, a recommender is validated by using evaluation metrics (for example, precision\@*k*, recall\@*k*, [MAP][map], [nDCG\@k][ndcg]).
```

Comment:
Instead of "is often used", let's provide prescriptive guidance.  Should I use the DSVM or not?

Reply:
let's take out "often"